src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -240,15 +274,81 @@ StaticLLMPipeline::StaticLLMPipeline(
     m_kvcache_request = core.compile_model(
         m_kvcache_model, device, generate_config
     ).create_infer_request();
-    // (8) Initialize tensors
-    prepare_for_new_conversation();
-};
+}
 
-StaticLLMPipeline::StaticLLMPipeline(
+void StaticLLMPipeline::setupAndImportModels(
     const std::filesystem::path& path,
     const std::string& device,
-    const ov::AnyMap& config
-) : StaticLLMPipeline(path, path.string(), device, config) {
+    ov::AnyMap& pipeline_config) {
+    /* To initialize pipeline in case when user passes "USE_BLOBS=YES",
+       next steps are required:
+        1) Check that neither MAX_PROMPT_LEN nor MIN_RESPONSE_LEN is
+           exposed in the config. These parameters will be retrieved
+           from blobs
+        2) Import prefill model from model directory or specified path
+        3) Import generate model from model directory or specified path
+        4) Fill in m_kvcache_desc
+    */
+    ov::Core core;
+
+    auto prefill_config = pop_or_default(pipeline_config, "PREFILL_CONFIG", ov::AnyMap());
+    auto generate_config = pop_or_default(pipeline_config, "GENERATE_CONFIG", ov::AnyMap());
+
+    auto import_blob = [this, 
+                        &path,
+                        &pipeline_config,
+                        &core,
+                        &device](const std::string& model_name,
+                                 ov::AnyMap& model_config) {
+        auto blob_path = pop_or_default(model_config, "BLOB_PATH", std::string{});
+
+        if (blob_path.empty()) {
+            blob_path = (path /
+                (std::string("openvino_") + model_name + ".blob")).string();
+        }
+
+        if (!std::filesystem::exists(blob_path)) {
+            OPENVINO_THROW("Blob for " + model_name + " model is not found at: "
+                + blob_path);
+        }
+
+        merge_config_with(model_config, pipeline_config);
+
+        std::fstream fs(blob_path, std::ios::in | std::ios::binary);
+
+        return core.import_model(
+            fs, device, model_config);
+
+    };
+
+    auto get_kvcache_size = [](ov::CompiledModel& model) {
+        for (auto input : model.inputs()) {
+            const auto& input_name = input.get_any_name();
+            if (input_name.find("attention_mask") != std::string::npos) {
+                return static_cast<uint32_t>(input.get_shape()[1]);
+            }
+        }
+        return uint32_t(0);
```

Comment:
I believe it's impossible situation, we can't proceed if there is no `attention_mask` in model. I'd suggest assert / exception here instead of `0`

Reply:
Fixed, thanks!